python/tests/test_json.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+import json
+
+import src.file_io
+import src.netlist
+import src.json
+
+JSON_MOCK_FILE = r"""{"__MockFile__": true, "_path": "/path/to/file", "_contents": ""}"""
+
+JSON_BASE_NETLIST_FILE = r"""{
+    "__BaseNetlistFile__": true,
+    "_path": "/path/to/a/different/file",
+    "_contents": ""
+}"""
+
+JSON_NETLIST = r"""{
+    "__Netlist__": true,
+    "base_netlist_file": {
+        "__BaseNetlistFile__": true,
+        "_path": "",
+        "_contents": ""
+    },
+    "cell_name": "Cell Name",
+    "device_widths": [
+        1,
+        3.0,
+        5.5
+    ],
+    "device_lengths": [
+        0
+    ],
+    "device_fingers": [
+        1000000000000000.0,
+        1e+16,
+        1e+16,
+        1.11e-12,
+        2e+19
+    ]
+}"""
+
+
+class TestJSON(unittest.TestCase):
+    def test_json(self):
+        self.maxDiff = None
+
+        file1: src.file_io.MockFile = src.file_io.MockFile(r"/path/to/file")
```

Comment:
can you use the `from src.file_io import MockFile` syntax to make this more readable?

Reply:
Same for others